bin/cheribsdtest/Makefile

```diff
@@ -23,6 +23,10 @@ SUBDIR+=	purecap-benchmark \
 		purecap-benchmark-dynamic \
 		purecap-benchmark-dynamic-mt \
 		purecap-benchmark-mt
+
+.if ${MACHINE_CPUARCH} == "aarch64" && ${MACHINE_ABI:Mpurecap}
+SUBDIR+=	benchmark-mt-c18n
```

Comment:
Please also teach share/examples/cheribsdtest/run-many about this (installed to /usr/share/examples/cheribsdtest/run-many as a helpful script to run one or more cheribsdtest variants and filter out all lines other than the summary):

```diff
diff --git a/share/examples/cheribsdtest/run-many b/share/examples/cheribsdtest/run-many
index ffdcdf6f3da8..90cbd35e9fbf 100755
--- a/share/examples/cheribsdtest/run-many
+++ b/share/examples/cheribsdtest/run-many
@@ -45,7 +45,7 @@ if [ $# -eq 0 ]; then
 
 	case `uname -p` in
 	aarch64c)
-		set -- "$@" c18n
+		set -- "$@" c18n benchmark-c18n
 		;;
 	esac
 fi
@@ -58,8 +58,8 @@ fi
 suffixes=
 for abi in "$@"; do
 	case "$abi" in
-	c18n)
-		suffixes="$suffixes mt-c18n"
+	c18n|benchmark-c18n)
+		suffixes="$suffixes ${abi%c18n}mt-c18n"
 		;;
 	*)
 		for dyn in '' -dynamic; do
```
should work, I believe